source/form.html

```diff
@@ -0,0 +1,121 @@
+          </li>
+          <li class="nav-list__item">
+            <a class="nav-list__link" href="photo.html">
+              Фото и видео
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a class="nav-list__link" href="">
+              Форма отзыва
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a class="nav-list__link" href="https://htmlacademy.ru/">
+              HTML Academy
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page-main">
+        <h1 class="visually-hidden">Оставьте свой отзыв</h1>
+        <section class="slogan">
+            <div class="slogan__wrapper">
+              <!-- Картинка гор будет добавлена как background -->
+              <img src="../source/img/text-sedona.svg" alt="Sedona">
+            </div>
+        </section>
+        <section class="comment">
+            <header class="comment__header">
+                <h2 class="comment__title">Оставьте свой отзыв</h2>
+                <p class="comment__description">Помогите нашим отелям стать лучше! Оставьте отзыв о них, а также о посещенных вами достопримечательностях</p>
+            </header>
+            <form class="comment__form form" method="post" action="">
+                <fieldset class="form__fieldset">
+                    <legend class="form__legend">Представьтесь:</legend>
+                    <label class="form__label" for="user-name">Имя*:</label>
+                    <input class="form__input" type="text" id="user-name" name="name" placeholder="Петр">
+                    <label class="form__label" for="last-name">Фамилия*:</label>
+                    <input class="form__input" type="text" id="last-name" name="last-name" placeholder="Иванов">
+                    <label class="form__label" for="middle-name">Отчество:</label>
+                    <input class="form__input" type="text" id="middle-name" name="middle-name" placeholder="Александрович">
+                </fieldset>
+                <fieldset class="form__fieldset">
+                    <legend class="form__legend">Контактная информация:</legend>
+                    <label class="form__label" for="user-phone">Контактный телефон*:</label>
+                    <input class="form__input" type="tel" id="user-phone" name="phone" placeholder="Введите телефон">
+                    <label class="form__label" for="user-email">Электронная почта*:</label>
+                    <input class="form__input" type="email" id="user-email" name="email" placeholder="Введите e-mail">                    
+                </fieldset>
+                <fieldset class="form__fieldset">
+                    <legend class="form__legend">Ваше общее впечатление:</legend>
+                    <label class="form__label form__label--bold" for="pisitive">Скорее положительное</label>
+                    <input class="form__input" type="radio" id="positive" name="impression" checked>
```

Comment:
I would use more descriptive ids for inputs, smth like "imput-impression-positive". In a big project such simple names may lead to intersection of ids from differnt parts of the code. But its up to you either change it here or not. Not a big deal for now

Reply:
if it may, it will :)  Do it right now. Add input- to each id.